src/components/Icon.js

```diff
@@ -15,8 +15,19 @@ import { colors, normalize } from '../config';
 export const Icon = ({ xml, width, height, name, size, focused, iconColor, style, iconStyle }) => {
   const color = iconColor || (focused ? colors.accent : colors.primary);
 
+  const hitSlop = (normalize(44) - size ?? width) / 2;
+
+  //for testing
+  console.warn(xml ? 'hitslopSvg:' : 'hitslopFont:', hitSlop);
+
   return (
-    <View style={style} hitSlop={{ top: 12, bottom: 12 }}>
+    <View
+      style={style}
+      hitSlop={{
+        top: hitSlop,
+        bottom: hitSlop
+      }}
```

Comment:
what about horizontal hitslop?

Reply:
solved f90ab7f

Reply:
```ts
const hitSlop = (44 - (size ?? width)) / 2;

return (
    <View
      style={style}
      hitSlop={hitSlop > 0 ? {
        top: hitSlop,
        bottom: hitSlop,
        left: hitSlop,
        right: hitSlop
      } : undefined}
````

Reply:
done in d906981aa68e8a03252497b159ecff3df02be64f